sorter.lic

```diff
@@ -0,0 +1,285 @@
+=begin
```

Comment:
💡 Consider documenting the script on elanthipedia:
```
=begin
  Documentation: https://elanthipedia.play.net/Lich_script_development#sorter
=end
```

Reply:
Done